cloud/amazon/ec2_asg.py

```diff
@@ -226,428 +218,557 @@
     region: us-east-1
 '''
 
-import time
 import logging as log
-import traceback
```

Comment:
The module's failure modes depend on this to send back the full trace to the user.